tests/Integration/AbstractTechproductsTestCase.php

```diff
@@ -332,6 +333,62 @@ public function testEscapes(string $requestFormat)
         $this->assertCount(0, $result);
     }
 
+    /**
+     * @see https://github.com/solariumphp/solarium/issues/1104
+     *
+     * @dataProvider updateRequestFormatProvider
+     */
+    public function testPhraseQuery(string $requestFormat)
+    {
+        $phrase = "^The 17\" O'Conner && O`Series \n OR a || 1%2 \r\n book? \r \twhat \\ text // ok? end$";
```

Comment:
```suggestion
        $phrase = "^The 17\" O'Conner && O`Series \n OR a || 1%2 1~2 1*2 \n book? \r \twhat \\ text: }{ )( ][ - + // \n\r ok? end$";
```

I even added now also `{}()[]-+*` chars to this string.